tests/import_v10x/run.sh

```diff
@@ -75,7 +75,7 @@ function run() {
 
 	# check task config, just a simple match
 	run_dm_ctl_with_retry $WORK_DIR "127.0.0.1:$MASTER_PORT" \
-		"get-config task test --file $WORK_DIR/task.yaml" \
+		"config task test --dir $WORK_DIR/task.yaml" \
```

Comment:
here we changed to `--dir` but parameter is not a dir 🤔

Reply:
How about `--path` 🤔